cli_035_surface_temp_analysis/contents/src/__init__.py

```diff
@@ -235,48 +363,57 @@ def extract_subdata_by_date(nc_file, dtype, nodata, available_dates, target_date
             'compress':'lzw',
             'nodata':nodata
         }
-        # Set filename
+        # generate a name to save the tif file we will create from the netcdf file
         sub_tif = os.path.join(DATA_DIR,'{}.tif'.format(FILENAME.format(date=date)))
         logging.info(sub_tif)
-
-
-
+        # create tif file for the available date
         with rio.open(sub_tif, 'w', **profile) as dst:
             dst.write(data.astype(dtype), indexes=1)
+        # add the new tif files to the list of tifs
         sub_tifs.append(sub_tif)
 
+    # delete the instance of the ncCDF4 class from memory
     del nc
     return sub_tifs
 
 
 def processNewData(existing_dates):
-    '''fetch, process, upload, and clean new data'''
-    # 1. Determine which years to read from the netCDF file
-    target_dates = getNewTargetDates(existing_dates)
+    '''
+    fetch, process, upload, and clean new data
+    INPUT   existing_dates: list of dates we already have in GEE, in the format of the DATE_FORMAT variable (list of strings)
+    RETURN  assets: list of file names for netcdfs that have been downloaded (list of strings)
+    '''
+    # Get list of new dates we want to try to fetch data for
+    target_dates = getNewDates(existing_dates)
 
-    # 2. Fetch datafile
+    # Fetch new files
```

Comment:
just changing this because we only fetch one file
```suggestion
    # Fetch data file from source
```